coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -820,6 +820,12 @@ export default class Directors extends Mixins(DateMixin, ExternalMixin, EntityFi
 
             var directors = response.data.directors
 
+            const fieldSorter = (fields) => (a, b) => fields.map(o => {
+              return a['officer'][o] > b['officer'][o] ? 1 : a['officer'][o] < b['officer'][o] ? -1 : 0
+            }).reduce((p, n) => p || n, 0)
+
+            directors = directors.sort(fieldSorter(['lastName', 'firstName', 'middleName']))
+
```

Comment:
We may only use the function twice so far, but would it make sense to move it into a mixin of sorts to prevent duplicating it?

Reply:
Sure. Will do that

Reply:
Moved the method to a mixin and added some documentation